shared/cluster.go

```diff
@@ -633,3 +633,64 @@ func FetchToken(ip string) (string, error) {
 
 	return token, nil
 }
+
+func CreateSecret(secret, namespace string) error {
+	kubectl := fmt.Sprintf("kubectl --kubeconfig %s", KubeConfigFile)
+	if namespace == "" {
+		namespace = "default"
+	}
+	cmd := fmt.Sprintf("%s create secret generic %s -n %s --from-literal=mykey=mydata",
+		kubectl, secret, namespace)
+	createStdOut, err := RunCommandHost(cmd)
+	if err != nil {
+		return ReturnLogError("failed to create secret: \n%w", err)
+	}
+	if strings.Contains(createStdOut, "failed to create secret") {
+		return ReturnLogError("failed to create secret: \n%w", err)
+	}
+	LogLevel("DEBUG", "Create Secret Output: %s", createStdOut)
+	return nil
```

Comment:
space before return, sorry idk why lint is not working on this, will take a look, but we have a clausule for that there

Reply:
fixed